### ISSUE TITLE
Add tuple() to exporter.

### ISSUE THREAD

**Issue by rredesigns** [godotengine/collada-exporter#47]
Hi there.

I modified the script to get rid of the error about endswith() expecting str or tuple, not frozen set. Changing this if condition in line 189:
if img_tmp_path.lower().endswith(bpy.path.extensions_image):

To:
if img_tmp_path.lower().endswith(tuple(bpy.path.extensions_image)):

Solves the problem perfectly. It's too little of a change to clone and make a pull request, so I will leave this little fix as an issue here. I exported a couple of times with no problems, I can't tell for sure if it will ever present some undesired behavior, but so far it does the job.

Thanks.

**Comment by akien-mga** [comment]
> It's too little of a change to clone and make a pull request, so I will leave this little fix as an issue here.

Now now, cloning and making a pull request takes as much time as writing out a patch in plain English on an issue as you did ;)

**Comment by rredesigns** [comment]
If I come up with another change I will do it the regular way next time. ;)

**Comment by bojidar-bg** [comment]
@rredesigns So, you think a PR with two unrelated changes is better than one which can be insta-merged?

By all means, just make a PR. You can even do it via the GitHub editor, not requiring you to do anything outside your browser.

**Comment by akien-mga** [comment]
> So, you think a PR with two unrelated changes is better than one which can be insta-merged?

I think they meant they would do a proper PR next time they have a patch proposal, but not for this very issue here :)

**Comment by rredesigns** [comment]
@bojidar-bg I completely overlooked that feature, tbh. Never used it, but that seems a practical way to contribute to a repo with minimal fixes. I'll check into that. Thanks. :D
